.github/workflows/cibuildwheel.yml

```diff
@@ -26,9 +36,26 @@ jobs:
         with:
           python-version: '3.7'
 
+      - uses: actions/cache@v2
+        id: numpy-aarch64-cache
+        with:
+          path: numpy-aarch64-cache/
+          key: numpy-aarch64-cache
+
       - name: Install cibuildwheel
         run: |
-          python -m pip install cibuildwheel==1.6.3
+          python -m pip install cibuildwheel==1.9.0
+
+      - name: Build numpy 1.16 for aarch64
+        if: matrix.cibw_archs == 'aarch64' && steps.numpy-aarch64-cache.outputs.cache-hit != 'true'
+        run: |
+          wget https://files.pythonhosted.org/packages/04/b6/d7faa70a3e3eac39f943cc6a6a64ce378259677de516bd899dd9eb8f9b32/numpy-1.16.0.zip
+          unzip numpy-1.16.0.zip
+          cd numpy-1.16.0
+          python -m cibuildwheel --output-dir ../numpy-aarch64-cache
```

Comment:
can we make this a hidden directory?  I assume that the folder needs to exist in the repo for the caching to work correctly?

Reply:
Ok. I prefer to have the wheels stored in a separate directory so as to not confuse pip and cibuildwheel when searching for wheels to install.

Reply:
That seems reasonable, but I do not want to clutter the top level of the repo with empty folders if we do not have.

Reply:
Understood. How about under the 'requirements' directory, seeing as numpy is a requirement for matplotlib?. I can make a hidden directory there.

Reply:
This is made in the build only, not checked in? It seems like `build/` or `dist/` would be a better location.

Reply:
I ran a few tests with cibuildwheel. There doesn't need to be any separate directory for the numpy wheels. cibuildwheel create a directory as part of its run. That should be sufficient.